src/resources/common.yml

```diff
@@ -123,6 +123,15 @@ sectionCopiedToClipboard:
 clipboardPasteNotSupported:
   en: Pasting from the clipboard is not supported by this browser.
   de: Das Einfügen aus der Zwischenablage wird von diesem Browser nicht unterstützt.
+copyUrlToClipboard:
+  en: Copy URL into the clipboard
```

Comment:
I would say "Copy ... to clipboard" sounds more natural than "Copy ... into the clipboard", in the English version. Sounds a bit translated from German to me 😋

Reply:
True, corrected (also in some old places)